### PR TITLE
ci(circleci): Migrate to CircleCI 2.0

### PR DESCRIPTION
**Pull request by obartra**
Closes: https://github.com/gymnastjs/gymnast/issues/377

### CODE REVIEW

**Comment by arahansen** [comment]
what are the image changes from?

File: scripts/citests.sh [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
-#
-# Splits test stage tasks between different CircleCI Nodes to improve performance.
-#
+mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
 
-echo "Running node $CIRCLE_NODE_INDEX"
+./scripts/deploystorybook.sh
+git checkout $CIRCLE_SHA1
 
-if [ "$CIRCLE_NODE_INDEX" -eq "0" ]; then
```
Comment (arahansen) [inline scripts/citests.sh]: is parallelization handled differently?

Comment (obartra) [reply scripts/citests.sh]: yup, it's gone. I didn't want to mess with it. this was simpler and similar time wise given picturebook 2.0 is faster

**Pull request merged**